tests/integration/rest/diddoc/metadata/negative_test.go

```diff
@@ -44,7 +44,7 @@ var _ = DescribeTable("Negative: Get DIDDoc version metadata", func(testCase uti
 			ExpectedResult: types.DidResolution{
 				Context: "",
 				ResolutionMetadata: types.ResolutionMetadata{
-					ContentType:     types.TEXT,
+					ContentType:     types.JSON,
```

Comment:
application/json is an accepted ContentType, not an Unaccepted type.

Reply:
TEXT is the unaccepted type, but the result is returned in json.

Similarly for Image/png. Current the error response is converted to an image.